### ISSUE TITLE
Shoulder labrum tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10994]
**Preferred term label:**
Shoulder labrum tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Glenoid labral tear; labral tear in the shoulder; Superior Labral Anterior Posterior tear (SLAP); bankart lesion; anterior labral periosteal sleeve avulsion (ALPSA); glenoid labral articular defect (GLAD)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder labrum tear, Shoulder dislocation, Shoulder internal impingement syndrome, Acromioclavicular (AC) joint injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001225
- label: Shoulder labrum tear
- parents: [Abnormal shoulder morphology (HP:0003043)](https://hpo.jax.org/app/browse/term/HP:0003043)
- pmid: 
- def: A injury with a team (rip, split, or disruption) to the labrum, which is the ring of cartilage that surrounds the socket of the shoulder joint (the glenoid).
- comment: Shoulder labral tears can be classified as superior Labral Anterior Posterior tear (SLAP); bankart lesion; anterior labral periosteal sleeve avulsion (ALPSA); and glenoid labral articular defect (GLAD) and may be observed in conditions such as shoulder dislocation, shoulder internal impingement syndrome, and acromioclavicular joint injury.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10994
- synonyms: Glenoid labral tear(NONE)|Labral tear in the shoulder(NONE)
